src/backend/base/langflow/components/vectorstores/mongodb_atlas.py

```diff
@@ -15,6 +18,15 @@ class MongoVectorStoreComponent(LCVectorStoreComponent):
 
     inputs = [
         SecretStrInput(name="mongodb_atlas_cluster_uri", display_name="MongoDB Atlas Cluster URI", required=True),
+        BoolInput(name="enable_mtls", display_name="Enable mTLS", value=False, required=True),
```

Comment:
What do you think of making this an advanced setting? 
```suggestion
        BoolInput(name="enable_mtls", display_name="Enable mTLS", value=False, advanced=True, required=True),
```